indico/modules/categories/client/js/components/CategoryStatistics.jsx

```diff
@@ -0,0 +1,195 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import categoryStatisticsURL from 'indico-url:categories.statistics_json';
+
+import moment from 'moment';
+import React, {useContext, useMemo} from 'react';
+import PropTypes from 'prop-types';
+import {Chart} from 'react-charts';
+import {Container, Header, Segment, Statistic, Message, Loader} from 'semantic-ui-react';
+import {Param, Translate} from 'indico/react/i18n';
+import {useIndicoAxios} from 'indico/react/hooks';
+import {LocaleContext} from '../index.js';
```

Comment:
don't import from `index.js` - either import from `../` (but I think that's quite ugly) or put it in a new `context.js` and then import it from `../context`

```js
import React from 'react';

const LocaleContext = React.createContext(null);
export default LocaleContext;
```